developer/src/tike/main/KeymanDeveloperOptions.pas

```diff
@@ -613,6 +619,11 @@ class function TKeymanDeveloperOptions.IsDefaultEditorTheme(s: string): Boolean;
   Result := DefaultEditorThemeItemIndex(s) >= 0;
 end;
 
+class function TKeymanDeveloperOptions.Default_DefaultProjectPath: string;
```

Comment:
A nit pick but I don't really like this function name. Reset_DefaultProjectPath is not right either because it doesn't actually reset it =. ln 373 does that. it is more `Get_Initial_DefaultProjectPath`  or `Get_Standard_DefaultProjectPath`